web/widgets.py

```
@@ -135,3 +135,268 @@ def __init__(self, attrs=None):
+                        if (newItem) {{
+                            newItem.querySelector('input').focus();
+                        }}
+                    }});
+
+                    // Initial render
+                    renderItems();
+                }});
+            </script>
+        </div>
+        """.format(
+            name, "".join(feature_items), json.dumps(decoded_value)  # {0}  # {1}  # {2}
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)